kubeflow/kubeflow
# feat(model-server): add Dockerfile of model-server with gpu support

Related to https://github.com/kubeflow/kubeflow/issues/194

### How to test

You could train a mnist model with https://github.com/tensorflow/serving/blob/master/tensorflow_serving/example/mnist_saved_model.py. 

Then mount the mnist saved model into tensorflow model server container and launch the gRPC server with mnist model.

Finally, you could launch a gRPC client with https://github.com/tensorflow/serving/blob/master/tensorflow_serving/example/mnist_client.py.

IMO, the saved model should not be added into the image. But we could put it into `model-server:1.4-example` image.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/kubeflow/kubeflow/210)
